zoho.go

```diff
@@ -68,7 +73,7 @@ func (z *Zoho) SetOrganizationID(orgID string) {
 type Zoho struct {
 	oauth OAuth
 
-	client         *http.Client
+	client         *retryablehttp.Client
```

Comment:
Can we keep this as `*http.Client`? Docs say you can expose get a [standard lib client with retries](https://github.com/hashicorp/go-retryablehttp#getting-a-stdlib-httpclient-with-retries)

This was if someone wants a modified `*http.Client` they can use `CustomHTTPClient`.